test/controllers/mobile_app/sensors_controller_test.rb

```diff
@@ -129,12 +129,13 @@ class SensorsControllerTest < ActionDispatch::IntegrationTest
     end
 
     test 'should show active sponsor of a sensor' do
-      create(:sponsor, active: true, sensors: [@sensor])
+      create(:sponsor, active: true, logo_source: 'hsr.svg', sensors: [@sensor])
```

Comment:
Just as a sidenote: The SVG is fine for testing, but at least on Android, rendering SVGs without using a WebView is not possible without including a large library. (SVG is quite a complex standard.) Therefore for production we probably have to stick to PNGs :slightly_frowning_face: